libraries/heft-config-file/src/ConfigurationFile.ts

```diff
@@ -59,6 +59,7 @@ interface IConfigurationFileFieldAnnotation<TField> {
 interface IConfigurationFileCacheEntry<TConfigurationFile> {
   configurationFile?: TConfigurationFile;
   error?: Error;
+  parentFileError?: Error;
```

Comment:
Possibly make this property the `parentConfigurationFileCacheEntry` for more utility?